guardrails-validation/README.md

```diff
@@ -43,27 +56,35 @@ Commit: 125160d
 Date: 2020-09-13T10:21:35Z
 ```
 
+7. Run the Tests
+
+If you are on a Mac or want to use a container skip this step
+```
+./run.sh # will run only the tests
+or
+./run-all.sh will run the configuration and download of the assets as well as run.sh
+```
 ### Container
 
 If you have docker installed you can build a container and run the tests through that.
 
+7. Build the container
+
+You will only need to do this step for the installation.
 ```
 docker build -t gc-guardrails:<tagname> .
 ```
 
-7. Run the Tests
+8. Run the Tests
 ```
-# Local
-./run.sh # will run only the tests
-or
-./run-all.sh will run the configuration and download of the assets as well as run.sh
-
-# Container
+# Run the newly built Container
 docker run -v $(pwd):/app gc-guardrails:<tagname>
 ```
 
 This will format the output from the inventory dump and run the tests. Results will be placed in the report.txt folder in the current directory.
 
+
+
```

Comment:
```suggestion
```

Reply:
@nschonni I pushed some changes last week and forgot to ping you. Can you take a look when you get a chance, thanks!